main.py

```diff
@@ -79,34 +67,29 @@ def print_card_special_options():
     player_blk = selected_card.block
     current_hand.pop(selected_card_index)
 
-    kos_atk, kos_blk, kos_mod = get_kos_clash_values()
-    kos_mod_message = get_mod_message(kos_mod)
-
-    kos_hp = clash(player_atk, player_blk, player, kos_atk, kos_blk, kos_hp)
-    if player_atk == kos_blk and (kos_mod == "Wolf" or kos_mod == "Star"):
-        player.receive_damage(1)
+    npc_atk, npc_blk, npc_mod = get_npc_clash_values()
+    npc_mod_message = get_mod_message(npc_mod)
 
-    kos_is_alive = kos_hp > 0
+    clash(player_atk, player_blk, player, npc_atk, npc_blk, npc_mod, npc)
 
     clash_number += 1
     print(f'\n== CLASH {clash_number} ==')
     print(f'\nPlayer atk: {player_atk}')
     print(f'Player blk: {player_blk}')
     print(f'Player mod: {selected_card_special} <!! Player mod is not yet factored into clashes. !!>')
     print(f'Player mod msg: {player_mod_message}')
-    print(f'\nKoS atk: {kos_atk}')
-    print(f'KoS blk: {kos_blk}')
-    print(f'KoS mod: {kos_mod}')
-    print(f'Kos mod msg: {kos_mod_message}')
+    print(f'\nNPC atk: {npc_atk}')
+    print(f'NPC blk: {npc_blk}')
+    print(f'NPC mod: {npc_mod}')
+    print(f'NPC mod msg: {npc_mod_message}')
     print(f'\nPlayer HP: {player.hp}')
-    print(f'Kos HP: {kos_hp}')
-
+    print(f'NPC HP: {npc.hp}')
 
 print("\n===== GAME OVER =====")
 if not player.is_alive:
-    if kos_is_alive:
+    if npc.is_alive:
```

Comment:
Probably sounds better to say "You are injured, and you have lost one rank in reputation."

Reply:
Updated.

Reply:
Resolve if happy with change.